### ISSUE TITLE
Cannot install rvm. gpg key install fails. Server receive failed

### ISSUE THREAD

**Issue by rithvik1705** [rvm/rvm#4404]
## Cannot install rvm gpg key.
Error pops up 'keyserver receive failed.No keyserver available' 

## Steps to reproduce

1. open terminal
2. paste gpg install key

![rvm error](https://user-images.githubusercontent.com/39231768/41191841-120013f8-6c13-11e8-8b71-5e604aa93abc.png)

**Comment by x-yuri** [comment]
Is it just me, or my comment was deleted...

Anyways, `keys.gnupg.net` is a pool of servers:

```
$ dig +noall +answer keys.gnupg.net
keys.gnupg.net.         59897   IN      CNAME   hkps.pool.sks-keyservers.net.
hkps.pool.sks-keyservers.net. 1381 IN   A       51.15.53.138
hkps.pool.sks-keyservers.net. 1381 IN   A       18.191.65.131
hkps.pool.sks-keyservers.net. 1381 IN   A       176.9.147.41
hkps.pool.sks-keyservers.net. 1381 IN   A       18.9.60.141
hkps.pool.sks-keyservers.net. 1381 IN   A       37.191.226.104
hkps.pool.sks-keyservers.net. 1381 IN   A       193.164.133.100
hkps.pool.sks-keyservers.net. 1381 IN   A       216.66.15.2
```

Not all of them work. And even not all of them work all the time. See for yourself:

```sh
while true; do
    for ip in $(dig +noall +answer keys.gnupg.net | tail +2 | awk '{print $NF}'); do
        gpg --delete-keys --yes --batch 409B6B1796C275462A1703113804BB82D39DC0E3 &> /dev/null
        gpg --keyserver hkp://"$ip" --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 2>&1 \
            && echo "+$ip"
            || echo "-$ip"
        sleep 1
    done
done
```

**Comment by x-yuri** [comment]
@hermanomark Does it fail under [`vagrant`][1]? Try the `CNAME`: `hkps.pool.sks-keyservers.net`.

[1]: https://github.com/protobox/protobox/issues/159#issuecomment-152840998

**Comment by hermanomark** [comment]
@x-yuri nevermind. I managed to fix the issue I followed this source: https://github.com/rvm/rvm/issues/4215

**Comment by pkuczynski** [comment]
We just released a new version 1.29.7 fixing this issue.

[Become a backer](https://opencollective.com/rvm#backer) and support us with a small donation to help us continue our activities. Thank you if you already one! 🙏
